app/audit/models/ClaimARefundAuditModel.scala

```diff
@@ -38,18 +38,20 @@ case class ClaimARefundAuditModel(balanceDetails: Option[BalanceDetails],
   private def getAvailableCredit: Double = {
     balanceDetails.flatMap(_.availableCredit) match {
       case Some(_) => balanceDetails.get.availableCredit.get.toDouble.abs
-      case (_) => 0.00
+      case _ => 0.00
     }
   }
 
   private def getCreditType(credit: (DocumentDetailWithDueDate, FinancialDetail)): String = {
     val creditType: Option[CreditType] = credit._2.getCreditType
     val isPayment: Boolean = credit._1.documentDetail.paymentLot.isDefined
+    val taxYearString = s"${credit._1.documentDetail.taxYear - 1} to ${credit._1.documentDetail.taxYear} tax year"
+
     (creditType, credit._1.dueDate) match {
       case (Some(MfaCreditType), _) => "Credit from HMRC adjustment"
       case (Some(CutOverCreditType), _) => "Credit from an earlier tax year"
       case (Some(BalancingChargeCreditType), _) => "Balancing charge credit"
-      case (Some(RepaymentInterest), _) => "Credit from repayment interest"
+      case (Some(RepaymentInterest), _) => s"Credit from repayment interest - $taxYearString"
```

Comment:
not covered by IT test 
![Screenshot 2024-07-05 at 11 03 01](https://github.com/hmrc/income-tax-view-change-frontend/assets/165788042/5811b5c0-6cd4-4ed8-a180-ab057775b4b1)